OpenEdXMobile/src/main/java/org/edx/mobile/view/EditUserProfileFragment.java

```diff
@@ -498,6 +502,25 @@ private static TextView createField(@NonNull LayoutInflater inflater, @NonNull V
             put("label", field.getLabel());
             put("value", formattedValue);
         }}));
+
+        // set content description when text is truncated. Truncated text can only be retrieved
+        // the view has been laid out. 
+        textView.getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
+            @Override
+            public void onGlobalLayout() {
+                if (Build.VERSION.SDK_INT < 16) {
+                    textView.getViewTreeObserver().removeGlobalOnLayoutListener(this);
+                } else {
+                    textView.getViewTreeObserver().removeOnGlobalLayoutListener(this);
+                }
+                String contentDescription = TextUtils.ellipsize(textView.getText(),
+                        textView.getPaint(),
+                        (float) textView.getWidth(),
+                        TextUtils.TruncateAt.END).toString();
+                textView.setContentDescription(contentDescription);
```

Comment:
We usually want talkback to readback all the text, but in the case of "About Me", the text could be really long.

I guess it doesn't make sense to have special treatment for this textview. The problem is that the "About me" view is not clear as a button for editing the "About me" section if Talkback says "About Me: really really really really long paragraphahahahahahahahahahahahahahahahhahahah. Double tap to activate"